app/src/HomePage.php

```diff
@@ -1,24 +1,9 @@
 <?php
 
-use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
-
 class HomePage extends Page
 {
     public function canEdit($member = null)
     {
         return false;
     }
-
-    private static $db = array(
-        'RightContent' => "HTMLText"
-    );
-
-    public function getCMSFields()
-    {
-        $fields = parent::getCMSFields();
-
-        $fields->addFieldToTab("Root.RightContent", new HTMLEditorField("RightContent", "Right Content"));
```

Comment:
Why did we get rid of the right content? I assume this is where the login and download buttons used to live. We shouldn't change the way content is presented - presumably someone designed that sidebar to be there intentionally.

Reply:
It doesn't match the startup theme. I don't want to make any changes to the startup theme since I'm not a designer, and also the content throughout the demo site say "This is the startup theme", so I don't think we should be customising it anyway

The right content is currently this

![image](https://github.com/user-attachments/assets/4d1b5c41-9ec0-47ef-9e7a-679166faeb4e)

We could easily just add these links into the home page content - do you want me to do that?